src/test/java/restAssured/PetStore.java

```diff
@@ -0,0 +1,100 @@
+package restAssured;
+
+import models.Category;
+import models.Order;
+import models.Pet;
+import models.Tag;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+import util.*;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class PetStore {
+    boolean petWasFound;
+
+    CreatePet createPet = new CreatePet();
```

Comment:
since these methods in these classes can be considered as utility methods. they never have states, never change. therefore, it's also a viable option to make them static and you can then simplify your test classes.
either statically importing them, or just referencing them through the class.